packages/cli/src/commands/setup/i18n/templates/storybook.preview.js.template

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react'
+import { I18nextProvider } from 'react-i18next'
+import i18n from 'web/src/i18n'
+
+export const globalTypes = {
+  locale: {
+    name: 'Locale',
+    description: 'Internationalization locale',
+    defaultValue: 'en',
+    toolbar: {
+      icon: 'globe',
+      items: [
+        { value: 'en', right: '🇺🇸', title: 'English' },
+        { value: 'fr', right: '🇫🇷', title: 'Français' },
+      ],
+    },
+  },
+}
+const withI18n = (StoryFn, context) => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Can you add a comment about why this is safe to do?

Reply:
Oh! I forget to remove this after refactoring